operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -15,6 +15,11 @@ metadata:
 spec:
   podManagementPolicy: {{ .Values.ingester.podManagementPolicy }}
   replicas: {{ $rolloutZone.replicas }}
+  {{- if and (semverCompare ">= 1.27.x" (include "mimir.kubeVersion" .)) (.Values.ingester.persistentVolume.enableStatefulSetAutoDeletePVC)  }}
```

Comment:
I realized that this feature is available since 1.23 - present in the [swaggger API docs for 1.23](https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.23/api/openapi-spec/swagger.json), but not for [1.22](https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.22/api/openapi-spec/swagger.json)

The API in 1.23 ([prettier link](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#statefulsetpersistentvolumeclaimretentionpolicy-v1-apps)) seems to be the same as the current one. Should we change this check to be 1.23.x?